Dockerfile

```diff
@@ -80,7 +81,7 @@ USER icuser
 WORKDIR /home/icuser/iCount_src
 
 RUN ../.icountenv/bin/pip install -e .
-RUN ../.icountenv/bin/pip install -e .[docs,tests]
+RUN ../.icountenv/bin/pip install -e .[docs,test]
```

Comment:
Please remind me why we need two statements, first 
`RUN ../.icountenv/bin/pip install -e .`
and then also:
`RUN ../.icountenv/bin/pip install -e .[docs,test]`
Could just the second line be used?

Reply:
You are right. No need for two statements.